.github/issue_template.md

```diff
@@ -0,0 +1,39 @@
+<!--
```

Comment:
What is the story with this text being in the comments? Is it going to show up in the new issue screen?